gamestonk_terminal/cryptocurrency/coingecko/pycoingecko_overview_controller.py

```diff
@@ -0,0 +1,267 @@
+"""Cryptocurrency Controller"""
+__docformat__ = "numpy"
+# pylint: disable=R0904, C0302
+import argparse
+import pandas as pd
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.cryptocurrency.coingecko import pycoingecko_view
+
+
+class GeckoController:
+
+    CHOICES = [
+        "help",
+        "q",
+        "quit",
+        "global",
```

Comment:
Some of these commands are missing a `print("")` at the end.

These are: global, coins, defi, top_defi, top_dex, top_nft, nft_market, exchanges, ex_rates, platforms, products, indexes, derivatives,